.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+.env
```

Comment:
Add ".DS_Store"  to ".gitignore" file so you don't need to delete ".DS_Store" every time you commit code.